sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsAudience.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary> Cloud audiences available for TextAnalytics. </summary>
+    public readonly partial struct TextAnalyticsAudience : IEquatable<TextAnalyticsAudience>
+    {
+        private readonly string _value;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TextAnalyticsAudience"/> object.
+        /// </summary>
+        /// <exception cref="ArgumentNullException"> <paramref name="value"/> is null. </exception>
```

Comment:
We should consider `<remarks>` that mention callers should prefer using one of the constant members over creating a new instance unless they have special needs for doing so.

Reply:
This is the first that I've used one, so I don't have context to comment.    However, looking at it from the customer-centric view in the TA scenario, I can see someone making the assumption they need to provide the URL to their TA instance, which would break things.